tests/scripts/test_test.py

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
copyright? I guess blank line is incorrect anyway.

Reply:
Removed the file, it was just for local testing.